openassessment/xblock/staff_area_mixin.py

```diff
@@ -164,7 +166,14 @@ def get_staff_path_and_context(self):
             })
 
         # Include whether or not staff grading step is enabled.
-        context['staff_assessment_required'] = "staff-assessment" in self.assessment_steps
+        staff_assessment_required = "staff-assessment" in self.assessment_steps
```

Comment:
We should add a check in one of the bok choy staff override test points that the staff grade tab doesn't appear after a staff override has been done (in a problem where staff grading is not required).